lib/components/atoms/Video/Video.mock.js

```diff
@@ -0,0 +1,15 @@
+const defaultConfig = {
+  width: '100%',
+  height: 'auto',
+  autoPlay: true,
+  controls: true,
+  loop: true,
+  muted: true,
+  poster: '',
+  src: 'http://techslides.com/demos/sample-videos/small.mp4',
+  type: 'video/mp4',
+  children: <track src="/" default kind="subtitles" srcLang="en" label="English" />,
```

Comment:
There could be more then one track and we need to achieve AA level if accessibility